.github/workflows/linter.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint Code Base
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          DEFAULT_BRANCH: true
```

Comment:
```suggestion
DEFAULT_BRANCH: main
```

Reply:
thx - fixed now.. Also updated VALIDATE_ALL_CODEBASE to false so that only changed files are checked.